internal/scheduler/nodedb/nodeiteration.go

```diff
@@ -0,0 +1,409 @@
+package nodedb
+
+import (
+	"bytes"
+	"container/heap"
+	"encoding/binary"
+	"fmt"
+
+	"github.com/hashicorp/go-memdb"
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
+
+	"github.com/armadaproject/armada/internal/scheduler/schedulerobjects"
+)
+
+const NodeDominantQueueWildcard = "*"
```

Comment:
Do we still need this now we no longer consider dominantQueues?

Reply:
Good catch. Deleted.